README.md

```diff
@@ -30,6 +30,8 @@ Here are the available configuration options:
 modules:
   - module: "synapse_invite_checker.InviteChecker"
     config:
+        api_base_prefix: "/_synapse/client/test", # Prefix to expose these endpoints under, optional, configure only if you know why you need to change it.
```

Comment:
Imo we really should have different options for the different APIs. And an option to disable the contact API, since that is removed in the next TIM version afaik.